alpha-build-machinery/Makefile

```diff
@@ -12,7 +12,7 @@ examples :=$(wildcard ./make/examples/*/Makefile.test)
 # Dynamic values are replaced with "<redacted_for_diff>" so we can do diff against checkout versions.
 define update-makefile-log
 mkdir -p "$(3)"
-$(MAKE) -j 1 -C "$(dir $(1))" -f "$(notdir $(1))" --no-print-directory --warn-undefined-variables $(2) 2>&1 | \
+set -o pipefail; $(MAKE) -j 1 -C "$(dir $(1))" -f "$(notdir $(1))" --no-print-directory --warn-undefined-variables $(2) 2>&1 | \
```

Comment:
how much do I love the default error behaviour of bash with pipes.

Reply:
who doesn't :)

Reply:
that `set` sets it forever, doesn't it? intentional?

Reply:
every make line has its own shell

Reply:
right